web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -128,6 +146,14 @@
         { shortcut: { key: 'D', ctrl: true }, onShortcut: () => deselectAllAssets() },
         { shortcut: { key: 's' }, onShortcut: () => onStackAssets() },
       );
+      // you're not supposed to manage your assets on the trash page: there's no button to manage it, then there's no reason to have shortcuts to do it.
+      if ($page.url.pathname !== AppRoute.TRASH) {
+        shortcuts.push({ shortcut: { key: 'a', shift: true }, onShortcut: onArchive });
+        // on some page, you can't favorite assets (like on `/parthner`)
```

Comment:
```suggestion
        // on some pages, you can't favorite assets (like on `/partner`)
```

Reply:
Where do you enforce this?

Reply:
With `isAllFavorite` != `null`

Reply:
Should that be called `isAllowFavorite` then? I assumed `isAllFavorite` would be true if all selected assets are already favorites

Reply:
This variable allows us to know two things: 
- know if the assets should be liked or unliked. 
- to find out if the assets can be liked

Reply:
Ah, so `isAllFavorite === true` means that they should be unliked (`=== false` means they should be liked), and if it's null they can't be liked?

Reply:
Yes, exactly